deploy/k8s.yaml

```diff
@@ -400,6 +400,13 @@ spec:
         - name: JUICEFS_CONFIG_PATH
           value: /var/lib/juicefs/config
         image: juicedata/juicefs-csi-driver:v0.13.2
+        lifecycle:
+          preStop:
+            exec:
+              command:
+              - /bin/sh
+              - -c
+              - rm -rf /registration/csi.juicefs.com-reg.sock /csi/csi.sock
```

Comment:
Do we really need `-rf`?

Reply:
You are right, `-rf` is not needed, clear it.